embulk-standards/src/main/java/org/embulk/standards/RenameFilterPlugin.java

```diff
@@ -53,4 +70,38 @@ public PageOutput open(TaskSource taskSource, Schema inputSchema,
     {
         return output;
     }
-}
\ No newline at end of file
+
+    private Schema applyRule(Object rule, Schema inputSchema) throws ConfigException
+    {
+        if (rule instanceof String) {
+            @SuppressWarnings("unchecked")
+            String ruleString = (String)rule;
+            switch (ruleString) {
+            case "lower_case_to_upper":
+                return applyLowerCaseToUpper(inputSchema);
+            case "upper_case_to_lower":
+                return applyUpperCaseToLower(inputSchema);
+            default:
+                throw new ConfigException("Renaming operator \"" +ruleString+ "\" is unknown");
+            }
+        } else {
+            throw new ConfigException("Non-string renaming operator is not supported");
+        }
+    }
+
+    private Schema applyUpperCaseToLower(Schema inputSchema) {
+        Schema.Builder builder = Schema.builder();
+        for (Column column : inputSchema.getColumns()) {
+            builder.add(column.getName().toLowerCase(), column.getType());
```

Comment:
type: timestamp column has a `format`. Probably it should be kept after rename filter through.